unicore/ask/service/responses_api.py

```diff
@@ -15,30 +17,50 @@ def get_response_object(request):
 
 
 def get_responses(request):
-    question_uuid = request.matchdict['question_uuid']
+    uuid = request.matchdict['option_uuid']
 
-    question = request.db.query(Question).get(question_uuid)
+    option = request.db.query(QuestionOption).get(uuid)
 
-    if question is None:
+    if option is None:
         raise NotFound
 
-    return question.responses.order_by('question_id')
+    return option.responses
+
+
+def get_option_object(request):
+    uuid = request.matchdict['option_uuid']
+    option = request.db.query(QuestionOption).get(uuid)
+
+    if option is None:
+        raise NotFound
+
+    return option
 
 
 @resource(
-    collection_path='/responses/{question_uuid}',
+    collection_path='/responses/{option_uuid}',
```

Comment:
I think this should be `/responses` and then either `question_id` or `option_id` needs to be present in the schema.